File: app/src/main/java/com/icure/kses/modoo/constant/Modoo_Constants.java
package com.icure.kses.modoo.constant;

import java.util.Random;
import java.util.concurrent.ExecutorService;
import java.util.concurrent.Executors;

public class Modoo_Constants {
    //AsyncTask 최대 스레드 개수
    public static final ExecutorService EXECUTOR = Executors.newFixedThreadPool(10);

    public static final int HTTP_POST = 1;
    public static final int HTTP_GET = 2;
    public static final int HTTP_MULTIPART = 0;

    public static final int HTTP_CONNECTION_TIMEOUT_BASE = 10000;
    public static final int HTTP_CONNECTION_TIMEOUT_EACH_FILE = 60000;

    private static final String LINE_FEED = "\r\n";
    private static final String CHARSET_UTF_8 = "UTF-8";
    private static final String CHARSET_EUC_KR = "EUC-KR";
    private static final String TWO_HYPHEN = "--";
    private final static String BOURDARY = String.format("%x", new Random().hashCode());

}

File: app/src/main/java/com/icure/kses/modoo/constant/ModooConstants.kt
package com.icure.kses.modoo.constant

import java.util.*
import java.util.concurrent.Executors

object ModooConstants {

    //로그인 모드
    const val BtoB = 0
    const val BtoC = 1

    //AsyncTask 최대 스레드 개수
    val EXECUTOR = Executors.newFixedThreadPool(10)
    const val HTTP_POST = 1
    const val HTTP_GET = 2
    const val HTTP_MULTIPART = 0
    const val HTTP_CONNECTION_TIMEOUT_BASE = 10000
    const val HTTP_CONNECTION_TIMEOUT_EACH_FILE = 60000
    const val LINE_FEED = "\r\n"
    const val CHARSET_UTF_8 = "UTF-8"
    const val CHARSET_EUC_KR = "EUC-KR"
    const val TWO_HYPHEN = "--"
    val BOURDARY = String.format("%x", Random().hashCode())

    //Google Analytics event id
    const val EVENT_ID_ITEM_DETAILS_ACTIVITY_1 = "ITEM_DETAILS_ACTIVITY_001"
    const val EVENT_NAME_CART_BTN_ITEM_DETAILS_ACTIVITY = "Add to cart button clicked"
}